### PR TITLE
UI-8649 - Integrate SCB migration into the core

### PR DESCRIPTION
**Pull request by antoinetissier**
See UI-8411, UI-8649.
Pulling changes from the relevant commits of the `scb-5.0.9` branch, which is actually used by SCB for their migration, into `main`.
I advise to review commit by commit.

Note that in particular, the latest versions of 5.0 and 5.1 are being pulled, which as a nicee side effect paves the way for a release of `atoti-ui-migration` depending on 5.1.8 and 5.0.28 when those are released.

### CODE REVIEW

**Comment by antoinetissier** [comment]
@Nouzbe note that the first commit is a cherry-pick from one that you authored.
In the original commit, there is a part that seemed unwarranted to me, so I didn't keep it while cherry-picking: https://github.com/activeviam/atoti-ui-migration/commit/ac05373181155e20142279cdc79d1d4708a42d30#diff-8b0b121c9477d397722a3c637e8a78a889e35160e6d44fdd0ec15f72e3da062eL188-L211
Let me know if I missed something.

File: __mocks__/clipboard-polyfill.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
[the remark this answers is not in the document]
Comment (Nouzbe) [reply __mocks__/clipboard-polyfill.js:1]: Could you add this information as a comment in this file?

File: jest.config.js [hunk 2]
```diff
@@ -19,6 +19,9 @@ const config = {
       "|",
     )}).+\\.(${extensions.join("|")})$`,
   ],
+  moduleNameMapper: {
+    "^(.*)/dist/__test_resources__$": "$1/__test_resources__",
+  },
```
[the remark this answers is not in the document]
Comment (Nouzbe) [reply jest.config.js:24]: Same here.

**Review by Nouzbe — APPROVED** [review]
Looks good 👍 Can you please take the following into account before merging?

**Pull request merged**